classify.py

```diff
@@ -0,0 +1,52 @@
+"""Classify chart as a chart type."""
+
+import os
+from base64 import decodebytes
+from io import BytesIO
+from skimage.color import rgba2rgb
+from skimage.transform import resize
+from skimage.io import imread
+from sklearn.preprocessing import LabelEncoder
+from tensorflow import keras
+from gramex.data import filter as gfilter
+
+
+OP = os.path
+FOLDER = OP.abspath(OP.dirname(__file__))
```

Comment:
I think `abspath` is redundant here. `dirname` already gives the absolute path.

Reply:
fixed via 727c457